lib/openapi-connector.js

```diff
@@ -163,8 +167,18 @@ OpenApiConnector.prototype.setupDataAccessObject = function() {
       // if (this.settings.remotingEnabled) {
       //   remoting.setRemoting.call(swaggerMethod, swaggerOp);
       // }
+      // Set findTodos()
       this.DataAccessObject[methodName] = swaggerMethod;
+      if (operationName !== operationId) {
```

Comment:
This is more like a question regarding the purpose of this PR: Given the findTodo operation, why do we want to expose it multiple times based on different conventions? 
```ts
"get": {
        "x-controller-name": "TodoController",
        "x-operation-name": "findTodos",
        "tags": [
          "TodoController"
        ],
        "operationId": "TodoController.findTodos"
}
```

we will have:

- findTodos()
- TodoController_findTodos()
- apis.TodoController.findTodos()

Reply:
The `apis.TodoController` allows `lb4 openapi --client` to generate separate service proxies for each tag (controller) and only expose operations for the given tag.

`x-operation-name` is a LoopBack extension to have a unique name within a controller. Before this PR, it only exposes:

- TodoController_findTodos()